views/surveys/view-survey-results.php

```diff
@@ -0,0 +1,238 @@
+		<div class="form-group">
+		<select class="form-control" id="classes" name="classes">
+			<option value="" disabled selected hidden>Class</option>
+			<option value="Class 1">Class 1</option>
+			<option value="Class 2">Class 2</option>
+			<option value="Class 3">Class 3</option>
+			<option value="Class 4">Class 4</option>
+			<option value="Class 5">Class 5</option>
+			<option value="Class 6">Class 6</option>
+		</select>
+		</div>
+
+		</div>
+		</div>
+		</div>
+		<!------------------------ This is the date selector ------------------------->
+		<div class="container" >
+				<div class="row justify-content-md-center">
+			<div class="col-sm col-lg-2">
+		<div class="form-group">
+		<select class="form-control" name="Month" id="month">
+		<option selected disabled="disabled" value="">Month</option>
+			<option value="01">January</option>
+			  <option value="02">February</option>
+			  <option value="03">March</option>
+			  <option value="04">April</option>
+			  <option value="05">May</option>
+			  <option value="06">June</option>
+			  <option value="07">July</option>
+			  <option value="08">August</option>
+			  <option value="09">September</option>
+			  <option value="10">October</option>
+			  <option value="11">November</option>
+			  <option value="12">December</option>
+		</select>
+		</div>
+		</div>
+		<div class="col-sm col-lg-2">
+		<div class="form-group">
+		<select class="form-control" name="Day" id="day">
+			<option selected disabled="disabled" value="">Day</option>
+			<option value="01">1</option>
+			  <option value="02">2</option>
+			  <option value="03">3</option>
+			  <option value="04">4</option>
+			  <option value="05">5</option>
+			  <option value="06">6</option>
+			  <option value="07">7</option>
+			  <option value="08">8</option>
+			  <option value="09">9</option>
+			  <option value="10">10</option>
+			  <option value="11">11</option>
+			  <option value="12">12</option>
+			  <option value="13">13</option>
+			  <option value="14">14</option>
+			  <option value="15">15</option>
+			  <option value="16">16</option>
+			  <option value="17">17</option>
+			  <option value="18">18</option>
+			  <option value="19">19</option>
+			  <option value="20">20</option>
+			  <option value="21">21</option>
+			  <option value="22">22</option>
+			  <option value="23">23</option>
+			  <option value="24">24</option>
+			  <option value="25">25</option>
+			  <option value="26">26</option>
+			  <option value="27">27</option>
+			  <option value="28">28</option>
+			  <option value="29">29</option>
+			  <option value="30">30</option>
+			  <option value="31">31</option>
+		</select>
+		</div>
+		</div>
+		<div class="col-sm col-lg-2">
+		<div class="form-group">
+		<select class="form-control" name="Year" id="year">
+		<option selected disabled="disabled" value="">Year</option>
+			<option value="2017">2017</option>
```

Comment:
Make this non-static. When 2018 happens we'll have to update this. Look at the reports team's code. They have a drop down with years in it.